### ISSUE TITLE
Unable to use 'Aggregate' button in scenario_damage and scenario_risk calculation

### ISSUE THREAD

**Issue by alejocaldeGEM** [gem/oq-irmt-qgis#799]
For the attached job_damage.ini and job_risk.ini, after OQ successfully finishes the calculation, using the plugin button to show 'Aggregate' damage or losses, total or by tag, as shown in this screenshot,  the plugin runs into an error.

![screenshot](https://user-images.githubusercontent.com/43442706/201112115-102b211e-939c-494d-9518-4b70cc1b7a49.png)

Full error trace message:

```
Traceback (most recent call last):
  File "C:\Users/acalderon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\svir\dialogs\drive_oq_engine_server_dialog.py", line 973, in 
    self.on_output_action_btn_clicked(
  File "C:\Users/acalderon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\svir\dialogs\drive_oq_engine_server_dialog.py", line 986, in on_output_action_btn_clicked
    self.viewer_dock.load_no_map_output(
  File "C:\Users/acalderon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\svir\dialogs\viewer_dock.py", line 753, in load_no_map_output
    self.load_avg_losses_rlzs_aggr(
[Cali_Excercise.zip](https://github.com/gem/oq-irmt-qgis/files/9981408/Cali_Excercise.zip)

  File "C:\Users/acalderon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\svir\dialogs\viewer_dock.py", line 869, in load_avg_losses_rlzs_aggr
    self._get_tags(session, hostname, calc_id, self.iface.messageBar(),
  File "C:\Users/acalderon/AppData/Roaming/QGIS/QGIS3\profiles\default/python/plugins\svir\dialogs\viewer_dock.py", line 846, in _get_tags
    tag_name, tag_value = tag.split('=')
ValueError: too many values to unpack (expected 2)
```

The tags have been declared in the file "Exposure_Cal.xml".

Thank you
Windows 10
OpenQuake 3.11.4
Plug-in 3.11.4 (same issue in 3.11.6)

**Comment by alejocaldeGEM** [comment]
[Cali_Excercise.zip](https://github.com/gem/oq-irmt-qgis/files/9981426/Cali_Excercise.zip)

**Comment by ptormene** [comment]
Fixed in f67d9a69
